packages/main/src/ToolbarPopoverButton.hbs

```diff
@@ -0,0 +1,12 @@
+<ui5-button
+	icon="{{this.icon}}"
+	design="{{this.design}}"
+	?disabled="{{this.disabled}}"
+	?hidden="{{this.hidden}}"
+	?priority="{{this.priority}}"
+	@click="{{this.press}}"
+	class="ui5-tb-popover-button ui5-tb-popover-item {{this.classList.value}}"
```

Comment:
`this.classList.value` refers to the native `classList` property right? Is the idea to transfer whatever classes the user passed to the abstract item to the physical `ui5-button`? This is strange, because these classes would exist on a different HTML document (we render the `ui5-button` in the shadow root, where the classes of the user don't apply)

Reply:
This "feature" was very useful for writing tests, so I decided to leave it like that. Basically we transfer the additional classed that user creates on the buttons, in order to select them later.